guess.js

```diff
@@ -9,7 +11,13 @@ if (packageData &&
     testDir = packageData.directories.test;
     pattern = testDir + ((testDir.lastIndexOf('/', 0) === 0) ? '' : '/') + '**/*.js';
 }
+try {
+  babelrc = JSON.parse(fs.readFileSync(path.join(process.cwd(), '.babelrc'), 'utf-8'));
+} catch (e) {
+  babelrc = {};
+}
```

Comment:
I suggest that

1. create `lib/resolve-babelrc.js`
2. move the above lines to `resolve-babelrc.js` as a module.
3. `resolveBabelrce` by use of the module

```js
var resolveBabelrce require("./lib/resolve-babelrc");
var babelrc = resolveBabelrce(process.cwd(), {});// location, defaultOptions
```

Babel has similar function, but it is private...

- [babel/resolve-rc.js at master · babel/babel](https://github.com/babel/babel/blob/master/src/babel/api/register/resolve-rc.js "babel/resolve-rc.js at master · babel/babel")

What do you think about this refactoring?